src/index-browser.ts

```diff
@@ -19,11 +19,13 @@ export * from './chain';
 export * from './utils/crypto';
 export * from './utils/keystore';
 export * from './utils/bytes';
-export * from './tx';
 export * from './tx/builder';
 export * from './tx/builder/helpers';
 export * from './tx/builder/constants';
-export * from './tx/builder/schema';
+// TODO: move to constants
```

Comment:
@davidyuk you may have missed this

Reply:
nope, it is definitely to be fixed later